### DISCUSSION TITLE
Stripe webhook subscription not working although site domain is set to non-localhost

### DISCUSSION THREAD

**Discussion by falko-burghausen** [saleor/saleor#13260] [Deployment] [answered]
Hi, 

We are trying to implement Stripe payment using the Stripe plugin. And we are stumbling upon the Webhook field, which does not get updated. As we understand, this happens automatically if and only if the site domain is **not** set to localhost, which is not the case for our Saleor instance: we've updated the site domain via mutation, if we run:
```
{
  shop {
    domain {
      url
      host
    }
  }
}
```
We get back the result with the updated host and url, as expected:
```
{
  "data": {
    "shop": {
      "domain": {
        "url": "http://saleor.example.org/",
        "host": "saleor.example.org"
      }
    }
  }
```
Are we missing something or what could be the reason that Saleor does not subscribe to the web hooks?
We also noticed that if creating a test checkout and using the mutation to get active payment gateways, Stripe is not listed, although it is set active – not sure if that has something to do with each other. 

Thanks for your help!

**Comment by falko-burghausen** [comment, marked as answer]
Okay, solved it:

As you can see above, by just setting the shop domain, Saleor internally obviously uses **http**://saleor.example.com as URL (take care of the HTTP protocol), also for the webhook in Stripe. Stripe does not accept non-HTTPS webhooks in production mode and therefore refuses to create the webhook. 

To make sure Saleor internally uses https, I set `ENABLE_SSL=True`, which is for some reason not documented in the [environments overview](https://docs.saleor.io/docs/3.x/setup/configuration). After doing this, the webhook at Stripe is created and seems to work. 

@Saleor team: any reason why this is not documented?

**Reply by korycins** [reply]
Hey @falko-burghausen, I don't think that there is any reason why the env is not documented. 
I see that there is a task already created for this: https://github.com/saleor/saleor-docs/issues/822.
I will prepare the GH issue to update the webhook configuration for Stripe plugin in saleor-docs
Thanks for reporting this

**Reply by korycins** [reply]
Here is a task for updating the docs for StripePlugin: https://github.com/saleor/saleor-docs/issues/826

Thanks for reporting this!
